.github/workflows/tests.yml

```diff
@@ -41,4 +41,4 @@ jobs:
     - name: Install dotnet-format
       run: dotnet tool install -g dotnet-format
     - name: Check with dotnet-format
-      run: dotnet format --check Meilisearch.sln
+      run: dotnet format --verify-no-changes Meilisearch.sln
```

Comment:
Do you know why this change now?

Reply:
I had the same problem here: https://github.com/meilisearch/meilisearch-dotnet/pull/233 and I "solved" by fixing the version of the action actions/setup-dotnet@v1.

Reply:
dotnet format has been integrated with .net 6 sdk. They replaced `check` by `verify-nochanges` in order to have names that better fit with dotnet cli.

Reply:
Actually, you should keep your change and close this one. Building and testing are done against 3.1.

Reply:
Ok, good to know, thanks for sharing the knowledge!